utils/argparser.py

```diff
@@ -1,13 +1,18 @@
+""" 
+    Collection of what arguments this project needs.
```

Comment:
`Collection of the arguments needed in the project.` - maybe sounds better?

Reply:
Sg.